README.md

```diff
@@ -8,63 +8,305 @@
   </a>
 </p>
 <p align="center">
+    <a href="https://github.com/fingerprintjs/fingerprintjs-pro-vue/actions/workflows/test.yml">
+    <img src="https://github.com/fingerprintjs/fingerprintjs-pro-vue/actions/workflows/test.yml/badge.svg" alt="Build status">
+  </a>
    <a href="https://github.com/fingerprintjs/fingerprintjs-pro-vue/actions/workflows/release.yml">
     <img src="https://github.com/fingerprintjs/fingerprintjs-pro-vue/actions/workflows/release.yml/badge.svg" alt="Release status">
    </a>
-<a href="https://github.com/fingerprintjs/fingerprintjs-pro-vue/actions/workflows/test.yml">
-    <img src="https://github.com/fingerprintjs/fingerprintjs-pro-vue/actions/workflows/test.yml/badge.svg" alt="Tests status">
+   <a href="https://www.npmjs.com/package/@fingerprintjs/fingerprintjs-pro-vue-v3">
+     <img src="https://img.shields.io/npm/v/@fingerprintjs/fingerprintjs-pro-vue-v3.svg" alt="Current NPM version">
```

Comment:
We need to remove spaces inside link tags to avoid formatting problems:
<img width="330" alt="image" src="https://github.com/fingerprintjs/fingerprintjs-pro-vue/assets/1009415/35e6d64e-dbe3-440a-bdb8-eeead1fcae65">